tests/test_clean.py

```diff
@@ -494,6 +511,11 @@ def test_attributes_list():
             '<a href="#example.com">foo</a>',
```

Comment:
This test is confusing (probably my fault) since it uses a domain-ish thing, but it's testing anchors.

Reply:
Agreed! Added another less confusing looking anchor test and comment on the domain-ish test.